test/BUILD.bazel

```diff
@@ -24,7 +24,7 @@ ts_library(
     deps = [
         "//test/proto:pizza_service_ts_proto",
         "//test/proto/common:delivery_person_ts_proto",
-        "@npm//@types/jasmine",
```

Comment:
Do these need to change? I'd prefer to be as specific as possible so it's easier to trim unused dependencies.

Reply:
Done